### DISCUSSION TITLE
2023-05-26 - Microsoft Fabric - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by lukecookssw** [SSWConsulting/SSW.Rules.Content#5614] [Chewing the Fat]
Hey team,

Let's talk about [this rule](https://ssw.com.au/rules/data-lake)

**Comment by wicksipedia** [comment]
In the realm of data so drastic,
A tool emerged, named Microsoft Fabric.
With AI reports it's quite elastic,
Analyzing data, it's so fantastic,
In insights gleaned, it's truly dramatic!

**Comment by SamBWagner** [comment]
I think it's rad 🎉. I'm not the biggest fan of the over focus on CoPilot as it kind of detracts from the coolness that is Fabric itself. I'm also keen to start the journey of learning more about data in a professional environment and get it 'in code'. 🤓

**Comment by tkapa** [comment]
very nice for people who are fine with being locked into the MS ecosystem. Like Sam said, I dont much care for the CoPilot side of this.

**Comment by ozdemir-mehmet** [comment]
Look forward to stitching the fabric

**Comment by calumjs** [comment]
Nice marketing vids - would love some clarification of what we can do now that we couldn't do before though

**Reply by brydeno** [reply]
It's mainly nothing new (it's much more a stitching together of all the disparate pieces they had, probably why they call it fabric :) ).
The AI engine is even not that new, although probably getting towards being functionally usable now.

**Comment by jakebayliss** [comment]
Rule is do you use a data lake but mostly talks about Microsoft Fabric? I feel like it should focus on what a data lake is rather than what Fabric is, either that or change the rule title. Almost no info on what a data lake is/why its used/pros/cons

**Reply by SamBWagner** [reply]
Very true. Should be called "Data Lakes - Do you use Microsoft Fabric"

**Reply by brydeno** [reply]
I think the problem is a Data Lake is one component of Fabric, and not even the most important. Data Lakes are just giant buckets of related files with data in them.... Fabric, in theory is mainly about what you put on top of that to actually make sense of that. 
Keeping things in a data lake organised and clean is a big job, and all these things will help. Having them altogether in one place, may help more than having them all separate (as they were before).

**Reply by Hona** [reply]
Yep I clicked into the rule thinking I could catch a break from AI. then I was pleasantly surprised that it wasn't.

**Comment by jaydenalchin** [comment]
The videos are nice!

![pulp-fiction-john-travolta](https://github.com/SSWConsulting/SSW.Rules.Content/assets/75593567/cdc58c29-bbc1-46fe-b0f4-07f20555ae2b)

**Reply by hveraus** [reply]
I think Microsoft is getting better and better at their presentations, those Fluent UI things are really really nice on the showcase videos, but when I see the actual product I'll be like your meme. 😅

**Reply by Hona** [reply]
Yep the wider usage of Fluent UI is very cool

**Comment by camillars** [comment]
+1 with Jayden & I prefer the 2nd video 😆

**Comment by MarcusIrmscherSSW** [comment]
With my limited understanding, Fabric seems like it will be a great tool for streamlining data management and make data analytics so much simpler.

**Comment by chrisschultzssw** [comment]
Seems cool - as a SysAdmin, it's nice to see products being "unified". Hopefully it means less things (silos) to manage.

**Comment by AshAnil** [comment]
Fabric + Copilot = Awesome! +1 with Chris. Unifying products means fewer things to manage. Enjoy the benefits!

**Comment by danielmackay** [comment]
I like the idea of Microsoft Fabic (although I do wished they stopped reusing the [same name](https://azure.microsoft.com/en-au/products/service-fabric) for different things.  Would love to learn a bit more about this (among many other things), and see it in action in a real world environment.

**Comment by ncn-ssw** [comment]
I want to play with it before forming any solid opinions on it. The training appears to be [here](https://learn.microsoft.com/en-us/training/paths/get-started-fabric/).

**Comment by drwharris** [comment]
They look to have done a pretty good job of combining everything into a cohesive environment.

**Comment by MattParkerDev** [comment]
Fabric looks cool. We should mention OneLake in the rule in combination with Fabric.

**Comment by matt-goldman** [comment]
Cool stuff. The second video is better as it shows what the product does rather than what it is or how it works, or what features it has. We should probably switch the videos on the rule.

**Comment by Levijj22** [comment]
Very cool! 
Can't wait to ask co-pilot questions just like in the video's i.e. "Show me the top sales over the last 24 months" and it automatically visualizes after just a question.

**Comment by vladislav-kir** [comment]
Fabric looks awesome!

The rule, however has a wrong name, in my opinion. It talks too little about Data Lakes. The focus of the rule seems to be Fabric, so the rule should be named so

**Comment by cvionnet** [comment]
Microsoft Fabric - "One Platform to rule them all, One Platform to connect them, One Platform to unite them all and in seamless integration bind them"🧙‍♂️

**Reply by 2fernandez** [reply]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/27670973/e8082076-afef-4f9a-b8b8-6b132bda4e3e)

**Comment by lukecookssw** [comment]
Looks cool

**Comment by jimmy1992abc** [comment]
For me, this could be the most exciting tool released in the past few years (maybe more important than GPT3.5). I can see the potential of MS Fabric to solve the existing data issues and they are widely applied to most businesses!
For me - I cannot wait to experience and learn how to manage Fabric
For SSW - Huge potential to get more project.

**Comment by AlexRothwell** [comment]
This will be so useful but as with everything, you'll need to consider whether it's appropriate for each use case, I could see it having significantly worse performance than typical database systems

On the Rule, it read like a marketing piece for Microsoft Fabric to me, supported by the idea of data lakes

**Comment by Ben0189** [comment]
Awesome software, I'm excited to explore how Microsoft Fabric can contribute to businesses.

**Comment by tombui99** [comment]
I learnt Data Warehouse back in university and had quite a big interest in it. This is definitely a very cool new tool to play with!

**Comment by stephanfakossw** [comment]
Looks cool - keen to experience what it can do...

**Comment by amankumarrr** [comment]
With Microsoft Fabric, writing queries and analyzing data would become a seamless and straightforward task.

**Comment by sethdaily** [comment]
Not my area of expertise, but I enjoy working with data and data analysis when I do. If a use case ever came up for me, I would be enthused to give Fabric a try. I may also experiment with it on my own time, if I can figure it out.

**Comment by AntPolkanov** [comment]
I like the first video more as it focuses on Fabric rather than Copilot. 

I thought that today's CTF will be about Copilots😄

**Comment by tomek-i** [comment]
In the realm of data, where insights reside,
Microsoft Fabric emerges with its stride.
A powerful tool to consolidate and bind,
Data unified, a solution we find.

One tool to rule them, in a seamless flow,
Transforming chaos into insights that glow.
With Microsoft Fabric, knowledge takes flight,
Unlocking wisdom, empowering our sight.

---

looks like an interesting tool and would like to play around with it and explore its capabilities.
Training AI models, pipelines all cool stuff 👍 

The second video felt more like an advertisement of Copilot - however I liked the music 💯

**Comment by Harry-Ross** [comment]
I much prefer the second video, as it highlights Microsoft's focus on Copilots - which I feel will have a much more monumental impact on way that everyone uses Microsoft services - especially Azure OpenAI Studio. However, I do think that Fabric seems to be a very useful tool for data and analytics, and could help out SSW in a major way.

**Comment by zacharykeeping** [comment]
Fabric looks very cool, and the CoPilot integration looks to make working with it very easy.

**Comment by NadeekaJK** [comment]
Like the fact that all data tools are in one place, not so much value in the Rule though as it merely states what it is. More like a blog than a "rule". I would expect good practices what to do, what not to do in a Rule's context.

**Comment by christoment** [comment]
Fabric sounds awesome!
Looking forward to see this implemented on a project.

However... Microsoft already have "Service Fabric" - they should up their naming game 👆

**Comment by KahaMason** [comment]
Fabric looks Awesome! 😄 Excited to see this in-use with projects and the potential business value it can provide.

**Reply by CoffeeBugHub** [reply]
Fabric sounds cool, feel like it will make the whole data visualization and data transformation process much easier in the future. Looking forward to seeing the real cases.

**Comment by jontygardner** [comment]
+ 1 to @Levijj22's discussion. This seems like it will be great at pooling all the data from the great expanse and giving one single source of truth.

**Comment by 2fernandez** [comment]
I ❤️ the branding and quality of videos. Microsoft has upped their game in that regard; the graphics look nice.
Now for them only to connect the dots and start naming things appropriately.

To the rest of it: 

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/27670973/5d18871e-5eb7-4606-a2fb-a48b64cc11a4)

**Comment by bahjatalaadelssw** [comment]
It's important to note that unstructured data can be valuable for organizations as it contains untapped insights and potential business value. Effectively harnessing and analyzing unstructured data can provide a more comprehensive understanding of customer sentiment, market trends, and other valuable information that may not be readily available in structured data alone. 🤖

**Comment by leehawkinsssw** [comment]
The rule is specific to Fabric and should be named accordingly. I'd like to see rules on data lakes from a more conceptual perspective, this one is too focused on Fabric and hence will quickly become outdated.

**Comment by steven0x51** [comment]
Cool product, I can't wait to dive into the data lake

**Comment by UlyssesMaclaren** [comment]
Would we always use this even if we only care about Power BI? Or is this only if we care about the larger picture data lake kinda stuff?

**Comment by Dhruv-0987** [comment]
The concept of Fabric is fantastic because dealing with unstructured data can be very difficult, as per the tool it self the video didn't much explain much about it, they should have given at least one example or a use case to demo what we can actually use it for.

**Comment by PennyWalker** [comment]
I think this is really cool. I'd love to know more about it to see how we can apply it more.

**Comment by KostaMadorsky** [comment]
Would be great to play with it :)

**Comment by leahy268** [comment]
Fabric looks great.  Can't wait to have a play.

**Comment by AnkyChe** [comment]
Love the 2nd video. Agree that the rule should be named about Fabric not data lake.

**Comment by Hona** [comment]
please give us a break from AI. The overload of AI CTFs is killing my interest in learning about this new tech, as an example

**Comment by Aibono1225** [comment]
First time learning about Fabric.
I am amazed by its ease of use and its potential business value. It seems awesome at the moment, and I am keen to learn and use it in future projects.

**Comment by Berber-Mama** [comment]
OMG, this is cool. It feels like when I got the copilot when creating a complex spreadsheet, I can be as good as Jimmy is (sorry Jimmy, you're the best)!

**Comment by JaneYangssw** [comment]
Delighted to have gained knowledge about Microsoft Fabric, and I look forward to incorporating this excellent tool more extensively in my work.

**Comment by jimmidier** [comment]
I actually like the idea of reusing terms within Microsoft ecosystem, so that one who's familiar with Microsoft products can infer what it does by just judging from the name. People with more confidence and understanding on the product will be more likely to become real users.

**Comment by KristenHu** [comment]
The video is really cool, I'd like to see how it works in an actual environment.

**Comment by johnxu16** [comment]
It is fascinating and terrifying to see that we can create data analytics with copilot.

**Comment by SofieHong** [comment]
Learned a new concept "unstructured data" and only realized today that videos and images belong to unstructured data. 🎬🖼️

Prefer the second video because it did a better job of presenting "what we can do" with more details and explanations than the first video. 📝

**Comment by apforsyth** [comment]
Looks fantastic but you gotta wonder how integrated it all is, really...  Only a non-biased super user will be able to tell you, know anyone who can answer that?

**Comment by tino-liu** [comment]
This is going to be super helpful for figuring out complex data fetches. Imagine needing to figure out tons of inner and outer joins. What a nightmare.

**Comment by AttackOnMorty** [comment]
Copilot is everywhere!

**Comment by jerryluossw** [comment]
Big data and AI are the future, it's more and more clear.

**Comment by duanxinhuan** [comment]
I think modify data is really easy with padans by a few lines of python code. Why fabric?

**Comment by PothieuG** [comment]
I had never heard of Fabric until today, but it seems extremely powerful, and the assistance provided by Copilot seems incredible !

**Comment by BrookJeynes** [comment]
Seems like a useful tool when you can't remember SQL syntax

**Comment by Marxoz** [comment]
Looks good if it can do what is says it can do… Will definitely open a lot of opportunities for companies, but I would like to see the speed first…. It almost sounds too good to be true.

**Comment by Freego1783** [comment]
Looks promising ! I feel like the Copilot integration will be awesome 😄

**Comment by TobyChurches** [comment]
Looks like a cool product, looking forward to giving it a try! 🤓

**Comment by jackreimers** [comment]
The videos are well put together but as someone who isn't super familiar with data products I'll need to do some further reading on the topic. Very keen to learn more about it!

**Comment by ozair-ashfaque** [comment]
It looks like a powerful tool

**Comment by jernejk** [comment]
I really like the idea of Microsoft Fabric and how it makes ML (at least from the video) a lot more accessible, especially the unstructured kind.

If the purpose of the rule is to promote the MS Fabric, then it does its job. Just not used to seeing the rules to be used like that.

**Comment by JackDevAU** [comment]
Fabric seems like a super powerful tool and I can't wait to try it out! The videos gave a good understanding of what it does and its potential!
